scripts/get_das_info.py

```diff
@@ -109,8 +113,8 @@ def convert_for_top_queries(data: dict) -> str:
 
 
 convert_functions = {
-    "convert_to_desired_structure": convert_to_desired_structure,
-    "convert_for_top_queries": convert_for_top_queries,
+    "convert_default": convert_default,
+    "convert_top": convert_top,
```

Comment:
This is just nitpicking, but wouldn't it be better to keep the keys of this dict (and hence the CLI options) as short as possible? Maybe we could go with just `default` and `top`? The function names themselves can stay as they are with `convert_` prepended.